src/data/cluster_situations_de.json

```diff
@@ -0,0 +1,89 @@
+{
+    "section-main": {
+      "headline": {
+        "title": "Clustersituationen",
+        "textblock": [
+          "Du kannst für folgende Situationen ein erhöhtes Ansteckungsrisiko annehmen.",
+          "Vermeide diese soweit möglich.",
+          "Wenn sie unvermeidbar sind, nutze die Corona-Warn-App, um im Falle einer nachträglich erkannten SARS-CoV-2-Infektionskette schnell gewarnt zu werden, oder selbst warnen zu können."
+        ]
+      }
+    },
+    "section-general-listing": {
+      "headline": {
+        "title": "Generell gilt",
+        "sub-headline": " (\"3 Gs (plus)\"):"
+      },
+      "content": {
+        "feature-list": [
+          {
+            "title": "In geschlossenen Räumlichkeiten",
+            "text": "(mit geringer Frischluftzufuhr)"
+          },
+          {
+            "title": "im _G_edränge",
+            "text": "(vieler Leute auf engem Raum)"
+          },
+          {
+            "title": "bei en_g_em Kontakt",
+            "text": "(Angesicht-zu-Angesicht)"
+          }
+        ]
+      },
+      "section-plus-listing": {
+        "headline": {
+          "title": "Plus laute(r):"
+        },
+        "content": {
+          "feature-list": [
+            {
+              "title": "Gespräche"
+            },
+            {
+              "title": "Gesang"
+            }
+          ]
+        }
+      },
+      "section-examples-listing": {
+        "headline": {
+          "title": "Beispiele",
+          "sub-headline": "Je mehr der oben genannten Eigenschaften eine Situation aufweist, desto höher kann das Ansteckungsrisiko angenommen werden."
+        },
+        "content": {
+          "feature-list": [
+            {
+              "title": "Bahn-, Bus- oder Flugreisen"
```

Comment:
`feature-list`s on other pages can have hyperlinks, right? Thus, it might be useful to link to sources for each entry.

Reply:
Done, see [below](https://github.com/corona-warn-app/cwa-website/pull/347#issuecomment-687819271)